src/app/services/actions/query-action-creators.ts

```diff
@@ -16,21 +15,24 @@ export function queryResponseError(response: object): IAction {
 }
 
 export function runQuery(url: string): Function {
-  const headers = { Authorization: 'Bearer {token:https://graph.microsoft.com/}' };
-
   return (dispatch: Function) => {
     const respHeaders: any = {};
-
-    return fetch(`https://proxy.apisandbox.msdn.microsoft.com/svc?url=${url}`, { headers })
+    let authToken = '{token:https://graph.microsoft.com/}';
+    let graphUrl = `https://proxy.apisandbox.msdn.microsoft.com/svc?url=${url}`;
+    const authenticated = localStorage.getItem('authenticated');
```

Comment:
What we are getting from local storage is the access token. We should call localStorage.getItem('accessToken')

Reply:
access token is in the authenticated object... as per the design spec

Reply:
What does the authenticated object contain? I suppose an authenticated user information? I think we should rename it to authenticatedUser.